errors/node-module-in-edge-runtime.md

```diff
@@ -1,39 +1,25 @@
-# Using Node.js module in Edge runtime
+# Using Node.js Modules in Edge Runtime
 
 #### Why This Error Occurred
 
-The code in your [Middleware][middleware] or your [Edge API routes][routes] is using a feature from Node.js runtime.
+The code in your [Middleware][middleware] or your [Edge API Routes][routes] is using a feature from Node.js runtime.
 
 However, the Edge Runtime does not support [Node.js APIs and globals][node-primitives].
 
 #### Possible Ways to Fix It
 
-When it runs in dev mode, your application will show in the console, and in your browser, which file is importing and using an unsupported module.
-This module must be avoided: either by not importing it, or by replacing it with a polyfill.
+When running Next.js locally with `next dev`, your application will show in the console, and in your browser, which file is importing and using an unsupported module. This module must be avoided: either by not importing it, or by replacing it with a polyfill.
 
-Please note your code can import Node.js modules **as long as they are not used**.
-The following example builds and works at runtime:
+For example, you might replace the Node.js `crypto` module with the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API).
 
-```ts
-import { NextResponse } from 'next/server'
-import { spawn } from 'child_process'
-
-export default async function middleware() {
-  if (process.version) {
-    spawn('ls', ['-lh'])
-  }
-  return NextResponse.next()
-}
-```
-
-Dynamic imports in unreachable code path is also supported.
+> **Note:** You can import Node.js modules as long as they are not used.
```

Comment:
Do we need to clarify why you would want to import something unused?

Reply:
Nope, removed.